test/test.cc

```diff
@@ -128,6 +128,12 @@ std::vector< testsweeper::routines_t > routines = {
     { "dev-axpy",         test_axpy_device,         Section::device_blas1   },
     { "",                 nullptr,                  Section::newline },
 
+    { "dev-dot",          test_dot_device,          Section::device_blas1   },
+    { "",                 nullptr,                  Section::newline },
+
+    { "dev-dotu",         test_dotu_device,         Section::device_blas1   },
+    { "",                 nullptr,                  Section::newline },
+
```

Comment:
Actually don't need the `Section::newline` lines for each BLAS-1 routine. Compare the CPU `Section::blas1` above.

Reply:
Should I remove all the `Section::newline` from the BLAS-1 routines, or just the one related to this PR and make a secondary once merged cleaning up this?